deep-learning/src/main/python/synapse/ml/dl/PredictionParams.py

```diff
@@ -4,7 +4,7 @@
 from pyspark.ml.param import Param, Params, TypeConverters
 
 
-class PredictionParams(Params):
+class HasLabelColParam(Params):
```

Comment:
nit: maybe just call this and others "HasLabelCol" to align with scala naming

Reply:
Scala already has `HasLabelCol`, will naming the same name confuse users? We only need this because we need snake_case :(